src/olympia/users/tasks.py

```diff
@@ -15,12 +14,8 @@
 def delete_photo(dst, **kw):
     task_log.info('[1@None] Deleting photo: %s.' % dst)
 
-    if not dst.startswith(user_media_path('userpics')):
```

Comment:
Why is it safe to remove this? Will a `dst` that doesn't match `MEDIA_ROOT` + `userpics` simply not be deleted now?